src/TurbulenceClosures/turbulence_closure_implementations/anisotropic_minimum_dissipation.jl

```diff
@@ -173,6 +200,7 @@ Return the diffusive flux divergence `∇ ⋅ (κ ∇ c)` for the turbulence
             + ∂zᵃᵃᶜ(i, j, k, grid, κ_∂z_c, κₑ, c, closure)
            )
 end
+=#
```

Comment:
Should this commented out function be nuked?

Reply:
yes! thank you for catching!